### ISSUE TITLE
"ExpressionEngine X.X.X" shouldn't turn red for all CP users

### ISSUE THREAD

**Issue by shbchk** [ExpressionEngine/ExpressionEngine#1746]
I believe that only Super Admins should get information about system updates

**Comment by robinsowell** [comment]
As someone who handles a lot of support, I'm torn on this one.  A surprising number of existing sites don't have superadmin in the cp.  Often, they don't even KNOW there are whole sections of the site they don't see.  I think it's good for them to be more aware that there are updates available and that those updates really matter.

I do get more panic emails going 'what's going on, do I need to upgrade immediately?!?!?!'.  And while the panic is bad, the idea that 'yes, you really need to start thinking about upgrading' is something they should hear.  It's not breaking anything or getting in the way, but it's a legit message that 'hey, security is important and you should stay on top of it'.

**Comment by shbchk** [comment]
There is a section in Role CP Access settings, where checkbox "Show system update notice" can appear :)

![de2121a9-5978-47b6-835a-6c1464b8d5a3](https://user-images.githubusercontent.com/12234507/150541245-e36cad8c-fc08-413e-9259-4e3e961c0066.png)

**Comment by shbchk** [comment]
>  it's a legit message

This message is legit only in one case - if person who sees it is actually able to do something about it. In all other cases if something turns red with no visible reason and on click there is no clues on what is going on, users really do begin panicking: _Is it my fault? What did I do wrong? Should I do something? What I should do?_ 

Since there is no way to build separate CP with full editing functionality just for users, I am planning to allow CP access to all users. I guess, there is no choice for me but to set display:none.

**Comment by robsonsobral** [comment]
In case the user isn't a super admin, couldn't the message explicit tells it?

> Hey, your system is outdated and there's an important update available! Please, bother the super-admin to update it as ASAP!

**Comment by intoeetive** [comment]
I agree with the sentiment that the users need to know that an update is available - on many installs superadmin accounts could be not used for several month, yet it's crucial to know that security update is available.

But I also can see where the frustration is coming from when you can't see WHY the badge is turning red or blue, as it's not directly telling the reason. We need to include message about update available (without actual ability to run update, as that's superadmin-only thing)

**Comment by shbchk** [comment]
I believe that this should be a matter of Roles and permissions.

**Comment by intoeetive** [comment]
@TomJaeger link to small discussion we had in internal Slack: https://packettide.slack.com/archives/C030Y8HG9L3/p1681476592899239
